### DISCUSSION TITLE
Use -s/-n/-v property with an attribute with spaces

### DISCUSSION THREAD

**Discussion by pablogarijo** [IBM/ELM-Python-Client#32] [Q&A]
How can I use the -s/-n/-v property with an attribute with spaces (F.E "Planned SW Release Version" or 'Satisfied By Architecture Element')?

I am getting the next error:

dcterms:title,Planned SW Release Version,dcterms:identifier,r
^
Expected one of:
* COMMA
* LBRACE
* DOT
* COLON

I have tried introducing the data with "", with '' and with both at the same time, but I keep getting errors.

oslcquery -J https://doorsng.grupoantolin.com:9443/ -U -P -p "" -C "SW_DALU" -s dcterms:title -F "SW_DALU_V3.0" -q "((oslc:instanceShape='SW Non-Functional Requirement') || (oslc:instanceShape='SW Functional Requirement'))" -n "Planned SW Release Version" -O result3.csv

this is my query

**Comment by barny** [comment]
See issue #33
